### PR TITLE
Release/2.3.3

### LINKED ISSUE #44: Release version 2.3.3
This issue is for tracking changes for the 2.3.3 release.  Target release date: **Wednesday, March 4th**.

### [Release steps](https://github.com/10up/simple-page-ordering/blob/develop/CONTRIBUTING.md#release-instructions)
- [x] Branch: Starting from `develop`, cut a release branch named `release/2.3.3` for your changes.
- [x] Version bump: Bump the version number in `package.json`, `readme.txt`, and `simple-page-ordering.php` if it does not already reflect the version being released.
- [x] Changelog: Add/update the changelog in `readme.txt` and `CHANGELOG.md`.
- [x] Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate
- [ ] Translations: Update the .pot file by running `npm run makepot`.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.gitattributes`.
- [x] Readme updates: Make any other readme changes as necessary. `CHANGELOG.md` and `README.md` are geared toward GitHub and `readme.txt` contains WordPress.org-specific content. The two are slightly different.
- [ ] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `master` (`git checkout master && git merge --no-ff develop`). `master` contains the stable development version.
- [ ] Push: Push your master branch to GitHub (e.g. `git push origin master`).
- [ ] Release: Create a [new release](https://github.com/10up/simple-page-ordering/releases/new), naming the tag and the release with the new version number, and targeting the `master` branch. Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the [2.3.3 milestone](https://github.com/10up/simple-page-ordering/milestone/4?closed=1).  The release should now appear under [releases](https://github.com/10up/simple-page-ordering/releases) and in the WordPress admin as an update as well.
- [ ] SVN: Wait for the [GitHub Action](https://github.com/10up/simple-page-ordering/actions) to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [ ] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/simple-page-ordering/. This may take a few minutes.
- [ ] Close milestone: Edit the [2.3.3 milestone](https://github.com/10up/simple-page-ordering/milestone/4) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description field`), then close the milestone.
- [ ] Punt incomplete items: If any open issues or PRs which were milestoned for `2.3.3` do not make it into the release, update their milestone to `2.3.4`, `2.4.0`, `3.0.0` or `Future Release`.

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change

Release and version bump to 2.3.3.

### Alternate Designs

none.

### Benefits

Properly updates plugin version number to alert consumers to `all the wonderful new things`.

### Possible Drawbacks

none identified.

### Verification Process

Verified manually via GitHub Desktop UI.

### Checklist:

- [X] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

Closes #44 

### Changelog Entry

n/a

### CODE REVIEW

File: .github/workflows/push-asset-readme-update.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  push:
+    branches:
+    - master
+jobs:
+  master:
+    name: Push to master
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: WordPress.org plugin asset/readme update
```
Comment (dinhtungdu) [inline .github/workflows/push-asset-readme-update.yml]: We don't commit `pot` file so we need to generate it before deploying to svn.

Comment (dinhtungdu) [reply .github/workflows/push-asset-readme-update.yml]: Note that our plugin on wp.org doesn't have a localization file atm.

Comment (jeffpaul) [reply .github/workflows/push-asset-readme-update.yml]: @dinhtungdu I added a POT file generation step to our release process, want to pick that up on your Monday and then progress through any remaining steps on this release?

Comment (dinhtungdu) [reply .github/workflows/push-asset-readme-update.yml]: Should we include it in our workflow? Eg:
```
 - uses: actions/checkout@master
    - name: Build
      run: |
        npm install
        ./node_modules/.bin/grunt
        ./node_modules/.bin/grunt i18n
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -63,15 +63,17 @@ Head to the Pages list table and drag and drop pages to change the order. Refres
...
-8. Git tag: Tag the release in Git and push the tag to GitHub. It should now appear under [releases](https://github.com/10up/simple-page-ordering/releases) there as well.
-
-<p align="center">
-<a href="http://10up.com/contact/"><img src="https://10updotcom-wpengine.s3.amazonaws.com/uploads/2016/10/10up-Github-Banner.png" width="850"></a>
-</p>
+1. Branch: Starting from `develop`, cut a release branch named `release/X.Y.Z` for your changes.
+2. Version bump: Bump the version number in `package.json`, `readme.txt`, and `simple-page-ordering.php` if it does not already reflect the version being released.
+3. Changelog: Add/update the changelog in `readme.txt` and `CHANGELOG.md`.
+4. Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
+5. Translations: Update the .pot file by running `npm run makepot`.
```
Comment (dinhtungdu) [inline CONTRIBUTING.md]: Currently, we haven't defined npm scripts yet.

File: .github/workflows/push-asset-readme-update.yml (hunk 1 above)
Comment (dinhtungdu) [reply .github/workflows/push-asset-readme-update.yml]: I've just found that w.org runs `wp i18n make-pot` automatically so we don't need to generate pot file here, resolved.

File: CONTRIBUTING.md (hunk 2 above)
Comment (dinhtungdu) [reply CONTRIBUTING.md]: @jeffpaul can you remove the pot file generation step or revert the last commit due to https://github.com/10up/simple-page-ordering/pull/45#discussion_r389207162

Comment (jeffpaul) [reply CONTRIBUTING.md]: _done_

**Pull request merged**